pkg/core/blockchain.go

```diff
@@ -184,6 +187,7 @@ func NewBlockchain(s storage.Store, cfg config.ProtocolConfiguration, log *zap.L
 
 		generationAmount:  genAmount,
 		decrementInterval: decrementInterval,
+		noBonusHeight:     defaultNoBonusHeight,
```

Comment:
Can we make it configurable? I'd rather have it in configuration file so that different network could have different heights.

Reply:
And some don't do this at all.